pkgs/development/tools/zigmod/default.nix

```
@@ -0,0 +1,92 @@
+{ lib
+, stdenv
+, callPackage
+, fetchFromGitHub
+, cacert
+, git
+, zig
+}:
+
+let
+  pname = "zigmod";
+  version = "83";
+
+  src = fetchFromGitHub {
+    owner = "nektro";
+    repo = "zigmod";
+    rev = "r${version}";
+    sha256 = "1sprzjylyjifqvakz8cm0ghmiqzvrsvkyhw49yy6qb3i2pi6jwiz";
+  };
+
+  postUnpack = ''
+    # Replace the dependency on a GitHub gist that has been deleted
+    sed -i 's#gist.github.com/nektro#github.com/nektro#g' $sourceRoot/deps.zig
+  '';
+
+  preBuild = ''
+    export XDG_CACHE_HOME="$(mktemp -d)"
+  '';
+
+  depsDir = "zig-cache/zigmod";
+
+  deps = stdenv.mkDerivation {
+    inherit src postUnpack preBuild;
+
+    name = "${pname}-deps-${version}.tar.gz";
```

```suggestion
    name = "${pname}-deps-${version}.tar.gz";
    inherit pname version src postUnpack preBuild;
```
this way we keep the file extension while also having pname+version

---

Oh, nice.